Site/SiteOlarkModule.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+require_once 'Site/SiteApplicationModule.php';
+
+/**
+ * Module for dislaying Olark live chat
+ *
+ * @package   Site
+ * @copyright 2015 silverorange
+ * @link      https://www.olark.com/api
+ */
+class SiteOlarkModule extends SiteApplicationModule
+{
+	// {{{ public properties
+
+	/**
+	 * Locale strings
+	 *
+	 * @var array
+	 * @see https://www.olark.com/help/languages
+	 */
+	public $locale_strings = array();
+
+	// }}}
+	// {{{ protected properties
+
+	/**
+	 * Olark identity
+	 *
+	 * @var string
+	 */
+	protected $identity;
+
+	// }}}
+	// {{{ public function init()
+
+	public function init()
+	{
+		$config = $this->app->getModule('SiteConfigModule');
+		$this->identity = $config->olark->identity;
```

Comment:
I copied this style from the `SiteAnalyticsModule`. I'm open to changing it though.